worldedit-core/src/main/java/com/sk89q/worldedit/command/RegionCommands.java

```diff
@@ -400,22 +400,25 @@ public int stack(Actor actor, World world, EditSession editSession, LocalSession
 
     @Command(
         name = "/regen",
-        desc = "Regenerates the contents of the selection",
-        descFooter = "This command might affect things outside the selection,\n" +
-            "if they are within the same chunk."
+        desc = "Regenerates the contents of the selection"
     )
     @CommandPermissions("worldedit.regen")
     @Logging(REGION)
     public void regenerateChunk(Actor actor, World world, LocalSession session,
             EditSession editSession, @Selection Region region) throws WorldEditException {
         Mask mask = session.getMask();
+        boolean success;
         try {
             session.setMask(null);
-            world.regenerate(region, editSession);
+            success = world.regenerate(region, editSession);
         } finally {
             session.setMask(mask);
         }
-        actor.print("Region regenerated.");
+        if (success) {
+            actor.print("Region regenerated.");
+        } else {
+            actor.print("Unable to regenerate chunks.");
```

Comment:
Shouldn't this be `printError`?

As to the message, I think it's fine, maybe add a "See console for details." sentence.

Reply:
the issue is that there isn't a console message on unsupported platforms (eg no adapter). could add one i guess?

Reply:
Yes, I would add one. I think that's the cleanest solution at the moment, I would prefer an exception but it would be weird to return a bool status _and_ throw exceptions.